lib/src/screens/send/parse_address_from_domain.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:cake_wallet/entities/openalias_record.dart';
+import 'package:cake_wallet/entities/unstoppable_domain_address.dart';
+import 'package:cake_wallet/src/widgets/alert_with_one_action.dart';
+import 'package:cake_wallet/utils/show_pop_up.dart';
+import 'package:flutter/material.dart';
+import 'package:cake_wallet/generated/i18n.dart';
+
+const topLevelDomain = 'crypto';
+
+Future<String> parseAddressFromDomain(
+    BuildContext context, String domain, String ticker) async {
+  try {
+    final name = domain.split('.').last;
+
+    if (name.contains(topLevelDomain)) {
```

Comment:
Handle case if `domain` variable will be like ".crypto" or just "crypto". So better to make something like:
```
 final domainParts = domain.split('.');
 final name = domainParts.last;
 
 if (domainParts.length > 1 && name.contains(topLevelDomain)) {
 ...
```